3scale/porta
# THREESCALE-10924: Remove duplicated config .yml files, keep only those in config/

**What this PR does / why we need it**:

We currently have multiple places where `.yml` configuration files are set:

- `config` - the default folder
- `config/examples` - these are not added to the container image, but for development they need to be copied to `config`
- `openshift/system/config` - these are copied to the container image by Dockerfile commands.

This causes some inconveniences:
- whenever the config changes, we need to update all the files
- we sometimes need to remember to copy the updated files from `example` to local versions again

This can be avoided by having a single set of configs. Normally, they would not collide, because we use `development` and `test` for development, and `production` typically is only used in the container image. And even if we wanted to run `production` locally, the values are normally configured through env variables, so it's the matter of adjusting your `.env` file.

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-10924

**Verification steps** 

See these comments:
https://github.com/3scale/porta/pull/3766#issuecomment-2079554409
https://github.com/3scale/porta/pull/3766#issuecomment-2082590997

**Special notes for your reviewer**:
